recipe/meta.yaml

```diff
@@ -26,10 +26,9 @@ requirements:
     - meson
     - ninja
     - pkg-config
-    - pthread-stubs         # [linux]
-    - xcb-proto             # [linux]
-    - xorg-renderproto      # [linux]
-    - xorg-xproto           # [linux]
+    # - pthread-stubs         # [linux]
+    # - xcb-proto             # [linux]
+    # - xorg-xorgproto        # [linux]
```

Comment:
```suggestion
```

If the logs look good to you @h-vetinari we can probably push this last cleanup and add automerge.